pyramid_transactional_celery/transactional_task.py

```diff
@@ -85,19 +88,15 @@ def tpc_abort(self, transaction):
 
 
 class TransactionalTask(Task):
-    """A task whose execution is delayed until after the current transaction.
-    """
-    abstract = True
-
-    def original_apply_async(self, *args, **kwargs):
-        """Shortcut method to reach real implementation of
-        celery.Task.apply_sync"""
-        return super(TransactionalTask, self).apply_async(*args, **kwargs)
+    """A task whose execution is delayed until the current transaction gets committed."""
 
     def apply_async(self, *args, **kwargs):
-        if self.app.conf.get('CELERY_ALWAYS_EAGER', False):
-            return super(TransactionalTask, self).apply_async(*args, **kwargs)
-        else:
+        retries_count = kwargs.get('retries', 0)
+        call_from_tpc_finish = kwargs.get('call_from_tpc_finish', None)
```

Comment:
Este novo param podia ser removido dos kwargs, para nem enviar para o celery, e evitar qualquer má-interpretação ou erro.
```suggestion
        call_from_tpc_finish = kwargs.pop('call_from_tpc_finish', None)
```